packages/run/test/commands/run.test.ts

```diff
@@ -11,7 +11,7 @@ describeOrSkip('@acceptance run', () => {
     return test
     .stdout()
     .do(() => {
-      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
This wasn't included in the original oclif migration PR, is it necessary?

Reply:
In short, yes. The ever-changing balance of how our dependencies get resolved led to this being an error.`ban-ts-ignore` doesn't exist in how eslint is now being resolved.